### DISCUSSION TITLE
Cannot set product price properly in JPY

### DISCUSSION THREAD

**Discussion by a-for-alex** [saleor/saleor#7928] [Q&A]
Hi guys, 
I am getting an error with setting a product price in JPY. I've changed the DEFAULT_CURENCY environment variable to JPY. 
But I cannot set a product price properly. For example, if I set a price to 100 Yen, the dashboard displays an Invalid Value in the price column. 
_This is the log message shows on the console after I clicked SAVE button at the bottom_
![unknown](https://user-images.githubusercontent.com/88753642/130706559-2b76f4b0-c276-48ca-94bc-ac085302ae1a.png)

However, if I set the price to 101 Yen, I can add the product.

Does anyone know what is the problem and how I can fix this?
I'm using saleor version 2.11 and docker to develop.

Thanks,

**Comment by karolkielecki** [comment]
@a-for-alex Hi! We recommend you using Saleor 3.0--as this is our priority. The beta of 3.0 introduces many new features, including multichannel w/ multicurrency, bug fixes + improvements. Would you mind trying with 3.0 and letting me know if the issue appears?
